pkg/crd/testdata/cronjob_types.go

```diff
@@ -154,13 +154,18 @@ type CronJobSpec struct {
 
 	// This tests that min/max properties work
 	MinMaxProperties MinMaxObject `json:"minMaxProperties,omitempty"`
+
+	// This is a new field that is unexported and should be ignored
+	testing string
```

Comment:
Can you also add an inlined field?

Reply:
:+1: Might be good to test inlined fields from both exported and unexported struct types.

```go
type unexportedStruct struct {
	Foo string `json:"foo"`
	bar string
}

type ExportedStruct struct {
	Baz string `json:"baz"`
	qux string
}
```

And then in this struct:
```go
...
	unexportedStruct `json:",inline"`
	ExportedStruct `json:",inline"`
...
```

With this, we would expect to see `foo` and `baz`?